prime-router/src/test/kotlin/metadata/MapperTests.kt

```diff
@@ -80,6 +82,178 @@ class MapperTests {
         assertThat(mapper.apply(lookupElement, args, elementAndValues).value).isEqualTo("y")
     }
 
+    private fun assertEq(
```

Comment:
Remove this function and just use the assertThat to be consistent with the rest of your tests

Reply:
The amount of active code rewrite, I feel, totally outweighs that consistency.  The risk for break is exceedingly high.